ofl/notosans/METADATA.pb

```diff
@@ -7,44 +8,36 @@ fonts {
   name: "Noto Sans"
   style: "normal"
   weight: 400
-  filename: "NotoSans-Regular.ttf"
-  post_script_name: "NotoSans"
-  full_name: "Noto Sans"
-  copyright: "Copyright 2012 Google Inc. All Rights Reserved."
+  filename: "NotoSans[wdth,wght].ttf"
+  post_script_name: "NotoSans-Regular"
+  full_name: "Noto Sans Regular"
+  copyright: "Copyright 2015 Google Inc. All Rights Reserved."
 }
 fonts {
   name: "Noto Sans"
   style: "italic"
   weight: 400
-  filename: "NotoSans-Italic.ttf"
+  filename: "NotoSans-Italic[wdth,wght].ttf"
   post_script_name: "NotoSans-Italic"
   full_name: "Noto Sans Italic"
-  copyright: "Copyright 2012 Google Inc. All Rights Reserved."
+  copyright: "Copyright 2015 Google Inc. All Rights Reserved."
 }
-fonts {
-  name: "Noto Sans"
-  style: "normal"
-  weight: 700
-  filename: "NotoSans-Bold.ttf"
-  post_script_name: "NotoSans-Bold"
-  full_name: "Noto Sans Bold"
-  copyright: "Copyright 2012 Google Inc. All Rights Reserved."
-}
-fonts {
-  name: "Noto Sans"
-  style: "italic"
-  weight: 700
-  filename: "NotoSans-BoldItalic.ttf"
-  post_script_name: "NotoSans-BoldItalic"
-  full_name: "Noto Sans Bold Italic"
-  copyright: "Copyright 2012 Google Inc. All Rights Reserved."
-}
-subsets: "menu"
 subsets: "cyrillic"
 subsets: "cyrillic-ext"
 subsets: "devanagari"
```

Comment:
Is the Devanagari merged in fresh?

Reply:
Devanagari is NOT merged; I'm not sure where this came from; deleted Devanagari from metadata